Provider fails to start when table user_nonce is already present

**Describe the bug**
  warnings.warn('SQLALCHEMY_TRACK_MODIFICATIONS adds significant overhead and will be disabled by default in the future.  Set it to True to suppress this warning.')
provider_1_9d83741413ec | [2021-02-22 21:14:50 +0000] [18] [ERROR] Exception in worker process
provider_1_9d83741413ec | Traceback (most recent call last):
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
provider_1_9d83741413ec |     self.dialect.do_execute(
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/default.py", line 608, in do_execute
provider_1_9d83741413ec |     cursor.execute(statement, parameters)
provider_1_9d83741413ec | sqlite3.OperationalError: table user_nonce already exists
provider_1_9d83741413ec | 
provider_1_9d83741413ec | The above exception was the direct cause of the following exception:
provider_1_9d83741413ec | 
provider_1_9d83741413ec | Traceback (most recent call last):
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/gunicorn/arbiter.py", line 583, in spawn_worker
provider_1_9d83741413ec |     worker.init_process()
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/gunicorn/workers/base.py", line 119, in init_process
provider_1_9d83741413ec |     self.load_wsgi()
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/gunicorn/workers/base.py", line 144, in load_wsgi
provider_1_9d83741413ec |     self.wsgi = self.app.wsgi()
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/gunicorn/app/base.py", line 67, in wsgi
provider_1_9d83741413ec |     self.callable = self.load()
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/gunicorn/app/wsgiapp.py", line 49, in load
provider_1_9d83741413ec |     return self.load_wsgiapp()
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/gunicorn/app/wsgiapp.py", line 39, in load_wsgiapp
provider_1_9d83741413ec |     return util.import_app(self.app_uri)
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/gunicorn/util.py", line 358, in import_app
provider_1_9d83741413ec |     mod = importlib.import_module(module)
provider_1_9d83741413ec |   File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
provider_1_9d83741413ec |     return _bootstrap._gcd_import(name[level:], package, level)
provider_1_9d83741413ec |   File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
provider_1_9d83741413ec |   File "<frozen importlib._bootstrap>", line 991, in _find_and_load
provider_1_9d83741413ec |   File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
provider_1_9d83741413ec |   File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
provider_1_9d83741413ec |   File "<frozen importlib._bootstrap_external>", line 783, in exec_module
provider_1_9d83741413ec |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
provider_1_9d83741413ec |   File "/ocean-provider/ocean_provider/run.py", line 11, in <module>
provider_1_9d83741413ec |     from ocean_provider.myapp import app
provider_1_9d83741413ec |   File "/ocean-provider/ocean_provider/myapp.py", line 31, in <module>
provider_1_9d83741413ec |     db.create_all()
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/flask_sqlalchemy/__init__.py", line 972, in create_all
provider_1_9d83741413ec |     self._execute_for_all_tables(app, bind, 'create_all')
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/flask_sqlalchemy/__init__.py", line 964, in _execute_for_all_tables
provider_1_9d83741413ec |     op(bind=self.get_engine(app, bind), **extra)
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/sql/schema.py", line 4663, in create_all
provider_1_9d83741413ec |     bind._run_visitor(
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 2095, in _run_visitor
provider_1_9d83741413ec |     conn._run_visitor(visitorcallable, element, **kwargs)
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1656, in _run_visitor
provider_1_9d83741413ec |     visitorcallable(self.dialect, self, **kwargs).traverse_single(element)
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/sql/visitors.py", line 145, in traverse_single
provider_1_9d83741413ec |     return meth(obj, **kw)
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/sql/ddl.py", line 783, in visit_metadata
provider_1_9d83741413ec |     self.traverse_single(
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/sql/visitors.py", line 145, in traverse_single
provider_1_9d83741413ec |     return meth(obj, **kw)
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/sql/ddl.py", line 827, in visit_table
provider_1_9d83741413ec |     self.connection.execute(
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1011, in execute
provider_1_9d83741413ec |     return meth(self, multiparams, params)
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/sql/ddl.py", line 72, in _execute_on_connection
provider_1_9d83741413ec |     return connection._execute_ddl(self, multiparams, params)
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1068, in _execute_ddl
provider_1_9d83741413ec |     ret = self._execute_context(
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1316, in _execute_context
provider_1_9d83741413ec |     self._handle_dbapi_exception(
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1510, in _handle_dbapi_exception
provider_1_9d83741413ec |     util.raise_(
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/util/compat.py", line 182, in raise_
provider_1_9d83741413ec |     raise exception
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/base.py", line 1276, in _execute_context
provider_1_9d83741413ec |     self.dialect.do_execute(
provider_1_9d83741413ec |   File "/usr/local/lib/python3.8/dist-packages/sqlalchemy/engine/default.py", line 608, in do_execute
provider_1_9d83741413ec |     cursor.execute(statement, parameters)
provider_1_9d83741413ec | sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) table user_nonce already exists
provider_1_9d83741413ec | [SQL: 
provider_1_9d83741413ec | CREATE TABLE user_nonce (
provider_1_9d83741413ec | 	address VARCHAR(255) NOT NULL, 
provider_1_9d83741413ec | 	nonce VARCHAR(255) NOT NULL, 
provider_1_9d83741413ec | 	PRIMARY KEY (address)
provider_1_9d83741413ec | )
provider_1_9d83741413ec | 
provider_1_9d83741413ec | ]
provider_1_9d83741413ec | (Background on this error at: http://sqlalche.me/e/13/e3q8)
provider_1_9d83741413ec | [2021-02-22 21:14:50 +0000] [18] [INFO] Worker exiting (pid: 18)

--- a/ocean_provider/myapp.py
+++ b/ocean_provider/myapp.py
@@ -2,12 +2,13 @@
 #  SPDX-License-Identifier: Apache-2.0
 
 import os
+import sqlite3
 from os.path import abspath, dirname
 
 from flask import Flask
 from flask_cors import CORS
 from flask_sieve import Sieve
-
+from ocean_provider.models import db as db_models  # noqa isort: skip
 from ocean_provider.utils.basics import get_config
 
 app = Flask(__name__)
@@ -19,13 +20,17 @@ if "CONFIG_FILE" in os.environ and os.environ["CONFIG_FILE"]:
 else:
     app.config["CONFIG_FILE"] = "config.ini"
 
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
 
 PROJECT_ROOT = dirname(dirname(abspath(__file__)))
 app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:////" + os.path.join(
     PROJECT_ROOT, "db", get_config().storage_path
 )
 
-from ocean_provider.models import db as db_models  # noqa isort: skip
 
 db = db_models
-db.create_all()
+
+try:
+    db.create_all()
+except sqlite3.OperationalError:
+    pass

--- a/ocean_provider/myapp.py
+++ b/ocean_provider/myapp.py
@@ -8,7 +8,6 @@ from os.path import abspath, dirname
 from flask import Flask
 from flask_cors import CORS
 from flask_sieve import Sieve
-from ocean_provider.models import db as db_models  # noqa isort: skip
 from ocean_provider.utils.basics import get_config
 
 app = Flask(__name__)
@@ -27,7 +26,7 @@ app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:////" + os.path.join(
     PROJECT_ROOT, "db", get_config().storage_path
 )
 
-
+from ocean_provider.models import db as db_models  # noqa isort: skip
 db = db_models
 
 try:

--- a/ocean_provider/myapp.py
+++ b/ocean_provider/myapp.py
@@ -4,7 +4,6 @@
 import os
 import sqlite3
 from os.path import abspath, dirname
-
 from flask import Flask
 from flask_cors import CORS
 from flask_sieve import Sieve

